.github/workflows/markdownlint.yml

```diff
@@ -13,8 +13,9 @@ jobs:
     - name: check out code
       uses: actions/checkout@v4
 
-    - name: install markdownlint-cli
-      run: sudo npm install -g markdownlint-cli
-
     - name: run markdownlint
-      run: markdownlint .
+      uses: DavidAnson/markdownlint-cli2-action@v14.0.0
```

Comment:
I have a general question - how do we decide when to trust something like this or not?

Reply:
I'd hope that this will be more secure. I assume that now that we're using an actual version rather than just blindly downloading the latest, that dependabot can help alert us of vulnerabilities